processing/src/main/java/io/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -1575,20 +1565,36 @@ public boolean putToKeyBuffer(RowBasedKey key, int idx)
       @Override
       public int getKeyBufferValueSize()
       {
-        return Long.BYTES;
+        return Long.BYTES + Byte.BYTES;
```

Comment:
Does this mean that new null handling unconditionally adds overhead in groupBy query processing?

Reply:
yes, we would take more bytes for both the case, kept it this way to avoid having to maintain two separate code branches for each of long/float/double serde helper

Reply:
done, refactored the null handling, Now we do not have any overhead for non-null case.